recipes/lue/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -e
+
+declare -a PLATFORM_OPTIONS=()
+
+if [ $(uname) == Linux ]; then
+  # Kindly guide to conda's OpenGL...
+  PLATFORM_OPTIONS+=("-D" "OpenGL_GL_PREFERENCE=LEGACY")
+  PLATFORM_OPTIONS+=("-D" "OPENGL_opengl_LIBRARY:PATH=${BUILD_PREFIX}/${HOST}/sysroot/usr/lib64/libGL.so")
+  PLATFORM_OPTIONS+=("-D" "OPENGL_gl_LIBRARY:PATH=${BUILD_PREFIX}/${HOST}/sysroot/usr/lib64/libGL.so")
+else
+  export MACOSX_DEPLOYMENT_TARGET=10.14
```

Comment:
Is this needed?

Reply:
For my local conda build yes, to avoid picking up the system OpenGL library. Seems not to be required within this environment, I removed it.

Reply:
I'm not sure why the Windows build now fails, the last changes only applied to build.sh ...

Reply:
The OpenGL related options are not required at all, I removed them entirely...